ui/src/index.js

```diff
@@ -42,11 +42,18 @@ import {HEARTBEAT_INTERVAL} from 'shared/constants'
 
 const rootNode = document.getElementById('react-root')
 
+let browserHistory
 const basepath = rootNode.dataset.basepath || ''
 window.basepath = basepath
-const browserHistory = useBasename(createHistory)({
-  basename: basepath, // basepath is written in when available by the URL prefixer middleware
-})
+if (basepath) {
```

Comment:
It isn't necessary to do the `if/else` block here, if `basepath` is falsey we're setting it to empty string on line 45.